client/src/components/Shared/Form/index.js

```diff
@@ -14,12 +16,96 @@ import {
   StyledInput,
   StyledTextarea,
   StyledImgCon,
-}
-  from './form.style';
+} from './form.style';
 
 class Form extends Component {
-  constructor(props) {
-    super(props);
+  state = {
+    isOpen: false,
+    selectedCat: null,
+    selected_itemType: '',
+    selected_colors: '',
+    selected_brands: '',
+    selected_condition: '',
+    selected_labelSize: '',
+    selected_age: '',
+    categories: [
+      'Item Type*',
+      'Color*',
+      'Brand',
+      'Other',
+      'Condition',
+      'Label size',
+      'Age',
+      'purchase price',
```

Comment:
Please go through and capitalise the first words: e.g. purchase price should be 'Purchase price'.